pages/resources/message.mdx

```diff
@@ -574,16 +574,13 @@ Clips are a special type of attachment that can be sent with a message. They are
 Clip [attachments](#attachment-object) have the [`CLIP` flag](#attachment-flags) set, and have the additional fields `clip_created_at`, `clip_participants`, and optionally `title` and `application`.
 
 An attachment can be sent as a clip by specifying the `is_clip`, `clip_created_at`, and `clip_participant_ids` fields, and optionally the `title` and `application_id` fields.
-For an attachment to be a valid clip, it must have a UUID appended to the end of the file in binary format. An example in hexadecimal format is `75 75 69 64 A1 C8 52 99 33 46 4D B8 88 F0 83 F5 7A 75 A5 EF`.
+For an attachment to be a valid clip, it must be an MP4 and have the clips UUID appended to the end of the file in binary format. In hexadecimal format this is `75 75 69 64 A1 C8 52 99 33 46 4D B8 88 F0 83 F5 7A 75 A5 EF`.
 This can be done with the following pseudocode:
 
 ```py
-import uuid
-
 with open("cat.mp4", "r+b") as file:
     file.seek(0, 2)
-    file.write(b"uuid")
-    file.write(uuid.uuid4().bytes)
+    file.write(b"\x75\x75\x69\x64\xA1\xC8\x52\x99\x33\x46\x4D\xB8\x88\xF0\x83\xF5\x7A\x75\xA5\xEF")
```

Comment:
mild nit but maybe make this convert the actual uuid to bytes? this looks a little horror